.github/workflows/kind.yaml

```diff
@@ -98,8 +98,7 @@ jobs:
         run: |
           cilium status
           kubectl get pods --all-namespaces -o wide
-          curl -sLO https://github.com/cilium/cilium-sysdump/releases/latest/download/cilium-sysdump.zip
-          python cilium-sysdump.zip --output cilium-sysdump-out
+          cilium sysdump --debug --output-filename cilium-sysdump-out
```

Comment:
Is there any specific reason for having `--debug` here and not in the others? Is this leftover from testing?

Reply:
I think it must be a leftover — I've removed it (cc @tklauser)!